libvast/vast/command.hpp

```diff
@@ -124,7 +124,7 @@ class command {
                           argument_iterator error_position,
                           argument_iterator begin, argument_iterator end) const;
 
-  std::string subcommand_error(argument_iterator error_position,
+  std::string subcommand_not_found(argument_iterator error_position,
```

Comment:
Indentation is off.

This function checks for `if (error_position != end)`. Looking at the name this should be a (documented!) precondition instead and only checked via assertion.

Reply:
Oh for some reason I overlooked this comment, sorry. So `subcommand_not_found` is still not a good name because it should only create a string which describes what went wrong.
What about `unkown_subcommand_error()` that follows the style of `parse_error`.